tests/kubernetes/utils.py

```diff
@@ -73,8 +74,11 @@ def has_secret(name, namespace="default"):
     try:
         api.read_namespaced_secret(name, namespace=namespace)
         return True
-    except:
-        return False
+    except ApiException as e:
+        if 'Reason: Not Found' in str(e):
```

Comment:
`if e.status == 404:` will be more robust against client library upgrades.

Reply:
Thanks, I missed that.